lib/html2text/version.rb

```diff
@@ -1,3 +1,3 @@
 class Html2Text
-  VERSION = "1.1.0"
+  VERSION = "2.0.0"
```

Comment:
Any idea if this is safe to do in ruby? I.E. is it like go where changing the major version actually requires code changes for the package manager to pick it up?

Reply:
That might be a question for @mkostick or @rafalyesware.

Reply:
I'd argue this is not a breaking change - or even a change at all in terms of version/changelog.  This is a change only the gem build process itself.  Consumers of the gem will quite possibly not be affected.

For reference, the dependencies/requirements for consumers of a Ruby gem are in the `.gemspec` file, which is published externally.  The `.ruby-version` file is just for local development and CI